lib/engine/round/auction.rb

```diff
@@ -20,6 +20,8 @@ def auction?
       def select_entities
         @game.players
       end
+
+      def finish_round; end
```

Comment:
Why does the base class need this? Unless I missed it, I don't see common code calling it.

Reply:
The base class needs it because it's called in bidbox_auction.rb finish_round.  bidbox_action is a module imported into both auction.rb and stock.rb.  In stock.rb, super has to be called - so it's in the bidbox_auction module.